src/libinjector/linux_private.h

```diff
@@ -0,0 +1,128 @@
+ * alternative license can be requested from the author to integrate       *
+ * DRAKVUF into proprietary applications and appliances.  Please email     *
+ * tamas.k.lengyel@gmail.com for further information.                      *
+ *                                                                         *
+ * If you have received a written license agreement or contract for        *
+ * Covered Software stating terms other than these, you may choose to use  *
+ * and redistribute Covered Software under those terms instead of these.   *
+ *                                                                         *
+ * Source is provided to this software because we believe users have a     *
+ * right to know exactly what a program is going to do before they run it. *
+ * This also allows you to audit the software for security holes.          *
+ *                                                                         *
+ * Source code also allows you to port DRAKVUF to new platforms, fix bugs, *
+ * and add new features.  You are highly encouraged to submit your changes *
+ * on https://github.com/tklengyel/drakvuf, or by other methods.           *
+ * By sending these changes, it is understood (unless you specify          *
+ * otherwise) that you are offering unlimited, non-exclusive right to      *
+ * reuse, modify, and relicense the code.  DRAKVUF will always be          *
+ * available Open Source, but this is important because the inability to   *
+ * relicense code has caused devastating problems for other Free Software  *
+ * projects (such as KDE and NASM).                                        *
+ * To specify special license conditions of your contributions, just say   *
+ * so when you send them.                                                  *
+ *                                                                         *
+ * This program is distributed in the hope that it will be useful, but     *
+ * WITHOUT ANY WARRANTY; without even the implied warranty of              *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the DRAKVUF   *
+ * license file for more details (it's in a COPYING file included with     *
+ * DRAKVUF, and also available from                                        *
+ * https://github.com/tklengyel/drakvuf/COPYING)                           *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef LIBINJECTOR_PRIVATE_H
+#define LIBINJECTOR_PRIVATE_H
+
+#ifdef DRAKVUF_DEBUG
+
+extern bool verbose;
+
+#define PRINT_DEBUG(args...) \
+    do { \
+        if(verbose) fprintf (stderr, args); \
+    } while (0)
+
+#else
+#define PRINT_DEBUG(args...) \
+    do {} while(0)
+#endif
+
+#define ARRAY_SIZE(arr) sizeof((arr)) / sizeof((arr)[0])
+
+#ifdef __clang_analyzer__
+#define vmi_free_unicode_str g_free
+#endif
+
+#endif
```

Comment:
I don't think this separate header is needed, looks to be the same for Windows and Linux

Reply:
I created a seperate header because windows offset are not used in Linux, due to that gcc was giving [-Werror=unused-variable] this warning.

Reply:
Then it would make more sense to have a `private.h` that has the common stuff, and then a separate `win_private.h` that has the offsets. Right now you are duplicated stuff which should be avoided.

Reply:
okay.